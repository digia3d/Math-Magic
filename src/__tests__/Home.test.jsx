import React from "react";
import renderer from 'react-test-renderer';
import Home from "../components/Home";
import '@testing-library/jest-dom';

describe('Test of Home page', () => {
  it('test if Home page is rendered', () => {
      const HomeTest = renderer
          .create(<Home />)
          .toJSON();
      expect(HomeTest).toMatchSnapshot();
  });
});