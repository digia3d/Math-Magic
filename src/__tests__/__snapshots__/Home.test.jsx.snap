// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Home page test if Home page is rendered 1`] = `
Array [
  <h2>
    Welcome to our page!
  </h2>,
  <section
    className="home-sec"
  >
    <p
      className="home-description"
    >
      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum ipsum tempore, ea soluta tenetur expedita dignissimos similique deleniti doloribus fugiat! Necessitatibus, sint doloribus deserunt voluptates impedit cumque alias, aperiam illo dolore explicabo iure culpa distinctio inventore perspiciatis sunt ipsam atque.Lorem ipsum dolor sit amet consectetur, adipi sicing elit. Earum ipsum tempore, ea soluta tenetur expedita dignissimos similique deleniti doloribus fugiat! Necessitatibus, sint doloribus deserunt voluptates impedit cumque alias, aperiam illo dolore explicabo iure culpa distinctio inventore perspiciatis sunt ipsam a tque.
    </p>
    <p
      className="home-description"
    >
      Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum ipsum tempore, ea soluta tenetur expedita dignissimos similique deleniti doloribus fugiat! Necessitatibus, sint doloribus deserunt voluptates impedit cumque alias, aperiam illo dolore explicabo iure culpa distinctio inventore perspiciatis sunt ipsam atque. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum ipsum tempore, ea soluta tenetur expedita dignissimos similique deleniti doloribus fugiat! Necessitatibus, sint doloribus deserunt voluptates impedit cumque alias, aperiam illo dolore explicabo iure culpa distinctio inventore perspiciatis sunt ipsam atque.
    </p>
  </section>,
]
`;
